src/components/QueryFileUploader/utils/getFileInfo.js

```diff
@@ -0,0 +1,11 @@
+import { getFileExtension } from '@folio/stripes-data-transfer-components';
+import { SUPPORTED_FILE_EXTENSIONS } from '../../../utils/constants';
+
+export const getFileInfo = file => {
+  const fileType = getFileExtension(file).substring(1);
```

Comment:
Slice is more natural for this sort of things. Please use it.
```suggestion
  const fileType = getFileExtension(file).slice(1);
```